Core/scripts/dirac-platform.py

```diff
@@ -27,17 +27,10 @@
 __RCSID__ = "$Id$"
 
 import sys
-import getopt
+import argparse
 
-cmdOpt = ('h', 'help', 'help doc string')
-
-optList, args = getopt.getopt(sys.argv[1:], cmdOpt[0], cmdOpt[1])
-for opt, value in optList:
-  if opt in ('-h', '--help'):
-    print(__doc__)
-    print('Options::\n\n')
-    print("  %s %s : %s" % (cmdOpt[0].ljust(3), cmdOpt[1].ljust(20), cmdOpt[2]))
-    sys.exit(0)
+parser = argparse.ArgumentParser(usage=__doc__)
```

Comment:
```suggestion
parser = argparse.ArgumentParser(description=__doc__,
                                 formatter_class=argparse.RawDescriptionHelpFormatter)
```

Reply:
with `usage=__doc__` you get at the 
```
usage:

```
at the beginning of the help message, which the command output parser turns into
```
usage::

foobar
```
Which gives you that error. By filling description intstead of usage we get
```
usage: dirac-platform [-h]

foobar
```

which is good enough. We also use the rawdescription helper to preserve the whitespace of the docs

Reply:
Thank you! Applied.